CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(CMAKE_CXX_COMPILER clang++)
 set(CMAKE_CXX_STANDARD 17)
 
 # Build options.
-option(USE_DRM "Select WAYLAND or DRM as the display backend type" OFF)
+option(BACKEND_TYPE "Select WAYLAND or DRM-GBM or DRM-EGLSTREAM or X11 as the display backend type" WAYLAND)
```

Comment:
Select WAYLAND, DRM-GBM, DRM-EGLSTREAM, or X11 as the display backend type

Reply:
I fixed it.